src/Components/ProposalCard/ProposalCard.test.tsx

```diff
@@ -26,6 +35,7 @@ const validProps: ProposalCardProps = {
   statusProps: {

```

Comment:
Can you reuse the existing ProposalStatus fixtures here? I just merged those changes, but you'll have to pull them in to fix the merge conflicts anyway.

Reply:
Sure. will update proposalCard tests with fixtures.